app/(onboard)/assessment.tsx

```
@@ -1,271 +1,360 @@
+    const currentAnswer = assessment.answers[currentQuestion.id]
 
     switch (currentQuestion.type) {
-      case QuestionType.TEXT:
-        return <TextQuestion {...questionProps} />;
-      case QuestionType.MULTIPLE_CHOICE:
-        return <MultipleChoiceQuestion {...questionProps} />;
       case QuestionType.SCALE:
-        return <ScaleQuestion {...questionProps} />;
+        // For SCALE questions, options is an array of strings like ["0-1", "1-3", "3-5", "5-10", "10+"]
+        // Render them as selectable option buttons similar to multiple choice
+        const selectedScale = currentAnswer || ""
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)